src/pybind/cephfs/c_cephfs.pxd

```diff
@@ -46,6 +46,7 @@ cdef extern from "cephfs/libcephfs.h" nogil:
     void ceph_shutdown(ceph_mount_info *cmount)
 
     int ceph_getaddrs(ceph_mount_info* cmount, char** addrs)
+    int64_t ceph_get_fs_cid(ceph_mount_info *cmount)
```

Comment:
```suggestion
    int64_t ceph_get_fscid(ceph_mount_info *cmount)
```

fscid == file system cluster id (common acronym in cephfs docs/api)

Reply:
Hmmm . However, libcephfs already has this interface:

```
extern "C" int64_t ceph_get_fs_cid(struct ceph_mount_info *cmount)
```

Reply:
oh, then crap. Keep it as is then.